config/routes.rb

```diff
@@ -25,6 +25,7 @@
   scope ":slug", module: "tenants", as: "tenant" do
     get "/" => "items#index"
     resources :items
+    get  '/dashboard' => 'dashboard#show'
```

Comment:
Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.